lib/linguist/languages.yml

```diff
@@ -1036,6 +1036,18 @@ Frege:
   - .fr
   tm_scope: source.haskell
   ace_mode: haskell
+  
+FreeMarker:
+  type: programming
+  color: "#0050b2"
+  group: Java
```

Comment:
@pchaigno Oh, no I don't. It's not really HTML either since FreeMarker can be used to output anything. I'll remove the group unless you think it should e HTML like some of the other templating languages.